client/source/DelphiLint.Server.pas

```diff
@@ -859,13 +860,15 @@ constructor TLintServerThread.Create;
 begin
   inherited;
   FLock := TMutex.Create;
+  FServerStartedEvent := TEvent.Create;
 end;
 
 //______________________________________________________________________________________________________________________
 
 function TLintServerThread.CreateNewServerInstance: ILintServer;
 begin
   Result := TLintServer.Create;
+  FServerStartedEvent.SetEvent;

```

Comment:
Is there a reason that this is here instead of at the bottom of `RefreshServer`?
Is it undesirable to signal that the server has started **before** `FServer` is assigned, or does it not matter?

Reply:
While it practically doesn't matter, because `FServer` exists behind a lock, it's probably best to play it safe and do that.